earth_enterprise/rpms/shared/lib/key-value.sh

```diff
@@ -0,0 +1,219 @@
+# Copyright 2017 the Open GEE Contributors.
```

Comment:
It would be a good idea to write some unit tests for the key/value store.

Reply:
Did you write this code or is this copied from somewhere?  If it's from somewhere else, we'll need to attribute it and state the license.

Reply:
It's my code.

Reply:
Unit tests for the key-value store seems like a good sub-task we can give to someone.  Would you like to write it, or should I do that?

Reply:
I can write the ticket.